recipe/meta.yaml

```diff
@@ -13,19 +13,24 @@ source:
     - 0001-add-version-var.patch
 
 build:
+  skip: True  # [py==312 or s390x or (py>39 and arm64)]
   number: 0
-  noarch: python
   entry_points:
     - lunar-find=lunarcalendar.command:find
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
+  build:
+    - patch  # [not win]
+    - m2-patch  # [win]
   host:
     - pip
-    - python >=3.7
+    - python
+    - wheel
   run:
-    - ephem >=3.7.5.3
-    - python >=3.7
+    - ephem >=3.7.5.3  # [not arm64]
+    - ephem 3.7.7  # [arm64]
```

Comment:
I think `ephem >=3.7.5.3` is fine. Just update ephem to 4.1.5.

Reply:
We already have ephem 4.1 but it's not supported for arm64

Reply:
ephem 4.1 is not supported for arm64
```
09/27/24 13:37:41  INFO   WARNING: Tests failed for lunarcalendar-ext-0.0.1-py310hca03da5_0.tar.bz2 - moving package to
                          /var/folders/k1/30mswbxs7r1g6zwn8y4fyt500000gp/T/abs_200ngjlppc/croot/broken
                   INFO   export
                          PREFIX=/var/folders/k1/30mswbxs7r1g6zwn8y4fyt500000gp/T/abs_200ngjlppc/croot/lunarcalendar-ex…
                   INFO   export
                          SRC_DIR=/var/folders/k1/30mswbxs7r1g6zwn8y4fyt500000gp/T/abs_200ngjlppc/croot/lunarcalendar-e…
                   INFO   import: 'lunarcalendar'
                   INFO   import: 'lunarcalendar'
                   INFO   ephem 4.1.2 is not supported on this platform
                   INFO   TESTS FAILED: lunarcalendar-ext-0.0.1-py310hca03da5_0.tar.bz2
                   ERROR  Command failed with exit code 1
```

Reply:
We already have ephem 4.1.2, do you have any source that states 4.1.5 will work for arm64?

Reply:
That's weird. For an unknown reason, `lunarcalendar-ext` throws an error on `osx-arm64` only:
`└─ ephem >=3.7.5.3,<4.*  does not exist (perhaps a typo or a missing channel).`

Reply:
Yeah, that's because ephem<4 only exists for python<3.10 - hence the python version restriction.

Reply:
Set `sephem >=3.7.5.3` without selectors and will build fine.
If you update ephem to 4.1.5 then you can drop  the `py==312` skipping rule. I tested `ephem 4.1.5` on my osx-arm64 locally and there are no issues.

Reply:
I've had this in build #3, it throws the error mentioned above